src/PrestaShopBundle/Resources/views/Admin/Product/form_combinations_bulk.html.twig

```diff
@@ -0,0 +1,70 @@
+{{ form_start(form, {'action': '#', 'method': 'GET'}) }}
+<div class="col-md-12">
+  <div class="col-md-2">
```

Comment:
I've changed, replacing the first "col-md-12" by "row". Thank you for noticing me :)